deployments/kubernetes/base/README.md

```diff
@@ -0,0 +1 @@
+# Kustomize Base Files
```

Comment:
@phudtran can you add some sentences to describe what this folder contains and what are they use for?